WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -22,13 +22,49 @@ class WordPress_Sniffs_WP_I18nSniff extends WordPress_Sniff {
 
 	/**
 	 * These Regexes copied from http://php.net/manual/en/function.sprintf.php#93552
+	 * and adjusted for better precision and updated specs.
 	 */
-	const SPRINTF_PLACEHOLDER_REGEX = '/(?:(?<!%)(%(?:[0-9]+\$)?[+-]?(?:[ 0]|\'.)?-?[0-9]*(?:\.[0-9]+)?[bcdeEufFgGosxX]))/';
+	const SPRINTF_PLACEHOLDER_REGEX = '/(?:
```

Comment:
Yes, there is (in both regexes).

> only recognize the optional padding character if there is also a width specifier in the placeholder

The change is that the space as a padding character is now only recognized as such if accompanied by a width specifier. Other padding characters are still recognized as normal even when there is no width specifier.

In detail:
Line 34 - The space as a padding character has been removed.
Line 38 - 42 - this is the new part where the space is now only recognized as a padding character if there is also a width specifier
Line 37 + 42: the specs mention an optional padding character which can be added to the precision specifier, this was previously unaccounted for. Was `(?:\.[0-9]+)?`, changed to account for the padding option here.